bgpd/bgp_vty.c

```diff
@@ -6413,11 +6413,16 @@ DEFUN (clear_bgp_ipv6_safi_prefix,
        "Clear bestpath and re-advertise\n"
        "IPv6 prefix\n")
 {
-	int idx_safi = 3;
-	int idx_ipv6_prefixlen = 5;
+	int idx_safi;
+	int idx_ipv6_prefix;
+	safi_t safi = SAFI_UNICAST;
+	char *prefix = argv_find(argv, argc, "X:X::X:X/M", &idx_ipv6_prefix) ?
```

Comment:
This is undefined behavior, you need to set `idx_ipv6_prefix = 0`. You need to do the same for `idx_safi`.

Reply:
ok